### PR TITLE
some quick svelte stuff

### PR DESCRIPTION
**Pull request by cfitz**

### CODE REVIEW

File: src/components/Search.svelte [hunk 1]
```diff
@@ -109,48 +109,35 @@
     ],
   };
 
-  export const compositions = itemsjs(catalog, configuration);
```
Comment (cfitz) [inline src/components/Search.svelte:112]: You only need to use "export" for properties you need to pass in, so there's really no point using that for something like this right now..

File: src/pages/search.astro [hunk 2]
```diff
@@ -4,5 +4,5 @@ import Search from "../components/Search.svelte";
 ---
 
 <BaseLayout title="Pianolatron">
-  <Search />
```
Comment (cfitz) [inline src/pages/search.astro:7]: This was probably your biggest problem right here...it's a funny one..
We are only using the client-side version of Svelte, so you need to indicate that here...you on:clicks were probably firing on the server-side, which itemjs probably has no idea about...

File: src/components/Search.svelte [hunk 3]
```diff
@@ -109,48 +109,35 @@
...
-      return results;
-    }
+  const textSearchCompositions = ( {target } ) => searchCompositons("composer_desc", target.value);
+	onMount(() => {
+    searchCompositons("composer_desc");
+    console.log(results);
   })
 
-
+  $: results;
```
Comment (cfitz) [inline src/components/Search.svelte:129]: ignore this! Pretty sure this isn't doing anything...

File: src/components/Search.svelte [hunk 4]
```diff
@@ -109,48 +109,35 @@
...
+  
 </script>
 
 <h1>Super Simple Sorted List</h1>
-  <input on:change={(e) => {
-    textSearchCompositons(e.target.val)
-  }} placeholder="Search" />
+  <input on:input={textSearchCompositions}
```
Comment (cfitz) [inline src/components/Search.svelte:134]: you could also just do this on:input={ (event) => testSearchCompositions(event) }

**Review by markpernotto — APPROVED** [review]

**Pull request merged**